pages/registration.page.js

```diff
@@ -23,6 +28,30 @@ class RegistrationPage extends BasePage {
         })();
     }
 
+    async registrationWithBadEmail(bademail) { //   UNsuccessfull registration scenario
```

Comment:
no need for all these methods `registrationWithBadEmail, registrationWithUsedEmail, registrationWithIncorrectPassword` please update and use `registration` method instead

Reply:
done